packages/web-twig/src/Resources/components/Accordion/Accordion.twig

```diff
@@ -1,14 +1,14 @@
 {# API #}
 {%- set props = props | default([]) -%}
 {%- set _class = props.class | default(null) -%}
-{%- set _id = props.id -%}
-{%- set _elementType = (props.elementType is defined) ? props.elementType : 'section' -%}
+{%- set _id = props.id | default(null) -%}
+{%- set _elementType = props.elementType | default('section') -%}
 
 {# Class names #}
 {%- set _rootClassName = _spiritClassPrefix ~ 'Accordion' -%}
 
 {# Attributes #}
-{%- set _idAttr = _id ? 'id=' ~ _id ~ '' : null -%}
+{%- set _idAttr = _id ? 'id="' ~ _id ~'"' : null -%}
```

Comment:
What is the problem with this? Everywhere in our code is `{%- set _idAttr = _id ? 'id=' ~ _id ~ '' : null -%}`.

Reply:
I originally had it without the double quotes, as I saw it exactly in Modal, for example, but as @adamkudrna  noted, even if it's not correct everywhere, there should be double quotes.

Reply:
So I returned it to the original so that it was the same everywhere